src/action/AsyncCommentActions.ts

```diff
@@ -15,13 +15,21 @@ import Comment, {
 } from "../model/Comment";
 import { ErrorData } from "../model/ErrorInfo";
 
-export function loadTermComments(termIri: IRI) {
+export function loadTermComments(
+  termIri: IRI,
+  dateTo?: string,
+  dateFrom?: string
+) {
   const action = { type: ActionType.LOAD_COMMENTS };
   return (dispatch: ThunkDispatch) => {
     dispatch(asyncActionRequest(action, true));
+    const reqParams: any = {};
+    reqParams.namespace = termIri.namespace;
+    if (dateTo) reqParams.to = dateTo;
+    if (dateFrom) reqParams.from = dateFrom;
```

Comment:
I think you can actually skip the `if`s, as params with undefined/nullish value will not be added to the request (see `paramsSerializer` in Ajax).

Reply:
You are right, I totally missed that. Added commit which removes the `if`s